test/utils.spec.ts

```diff
@@ -96,24 +99,27 @@ describe('utils', () => {
       Object.defineProperty(process, 'arch', {
         value: 'arm',
       });
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
       process.config.variables = {} as any;
       expect(getHostArch()).toEqual('armv7l');
     });
 
-    it('should return uname on arm 6', () => {
-      if (process.platform !== 'win32') {
+    if (process.platform !== 'win32') {
+      it('should return uname on arm 6', () => {
```

Comment:
:+1: on reversing this order

Just curious, was the linter smart enough to catch this, or did you spot it while adding the eslint-disable-next-line comments?

Reply:
IIRC the linter (`eslint-plugin-jest`) was smart enough, it didn't like conditional `expect`s.